src/Microsoft.AspNet.Authentication.OpenIdConnect/OpenIdConnectHandler.cs

```diff
@@ -669,7 +669,6 @@ public OpenIdConnectHandler(HttpClient backchannel)
             var responseMessage = await Backchannel.SendAsync(requestMessage);
             responseMessage.EnsureSuccessStatusCode();
             var userInfoResponse = await responseMessage.Content.ReadAsStringAsync();
-            var userInfoEndpointJwt = new JwtSecurityToken(userInfoResponse);
             var user = JObject.Parse(userInfoResponse);
```

Comment:
@Tratcher I am aware of any IDPs that returns jwt. For testing, I crafted up a JWT. If we are going for your solution, then it makes sense to remove that logic from protocol validator, otherwise it will be redundant. Instead, it can take the 'user' object.